src/mappings/mappingHandlers.ts

```diff
@@ -30,13 +30,14 @@ async function setAliases() {
 
 export async function handleEvent(event: CosmosEvent) {
     const blockHeight = BigInt(event.block.block.header.height);
+    const id = `${event.block.block.id}-${event.tx ? event.tx.hash : 'start_end'}-${event.idx}`
```

Comment:
Do we need `${event.tx ? event.tx.hash : 'start_end'}`? Is event.idx unique to the block?

Reply:
yeah, we don't need that